modules/ROOT/pages/install-aws.adoc

```diff
@@ -244,15 +247,11 @@ docker run -it -v $(pwd):/src -w /src/aws \
   -state=tf-data/rtf.tfstate
 ----
 +
-The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, 
-the Runtime Fabric updates to Active status in Runtime Manager.
+* The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, 
+the Runtime Fabric is shown in *Active* status in Runtime Manager.
```

Comment:
```suggestion
Runtime Fabric is shown in *Active* status in Runtime Manager.
```
I earlier changed Active to `Active`, believing that this was a syntax display, but if it is UI, then please change that edit to *Active*.